src/features/controls.ts

```diff
@@ -378,11 +392,11 @@ export const ControlsMixin = (ModelViewerElement:
         }
 
         [$onChange]({source}: ChangeEvent) {
-          this[$deferInteractionPrompt]();
           this[$updateAria]();
           this[$needsRender]();
 
           if (source === 'user-interaction') {
+            this[$deferInteractionPrompt]();
```

Comment:
Is this intentional? The implications are meaningful...

Reply:
Good catch; I must have changed this when I was trying to diagnose a test failure. Reverted.